lib/commentOnOpenedPullRequest.js

```diff
@@ -77,7 +77,6 @@ commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pull
                 errorCla: errorCla,
                 askAboutChanges: askAboutChanges,
                 askAboutContributors: askAboutContributors,
-                contributorsUrl: repositorySettings.contributorsUrl,
```

Comment:
Why are we not just updating the settings?

Reply:
Do you mean just set a new `contributorsUrl` config? Because I wanted the URL concierge links to to link to the correct branch. 

But now that I think about it, that won't work if the PR is a fork as opposed to a branch...

Maybe it's okay to just link to the fixed URL in master?

Reply:
Hmm, then I guess there'd need to be 2 contributor URLs in the settings: a GitHub API one and a regular one, which feels like not a great design.